src/sass/layout/modals/_franchise.scss

```diff
@@ -0,0 +1,242 @@
+  }
+
+  @include tablet {
+    width: 568px;
+    height: 668px;
+    padding: 32px 60px;
+  }
+}
+
+.franchise__button {
+  position: absolute;
+  left: 90%;
+  top: 1%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  background-color: inherit;
+  border: none;
+  color: getColor('theme-black');
+  cursor: pointer;
+  width: 20px;
+  height: 20px;
+
+  transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+  &:hover,
+  &:focus {
+    color: getColor('orange-main');
+  }
+
+  & .franchise__icon {
+    fill: currentColor;
+    width: 11px;
+    height: 11px;
+  }
+
+  @include tablet {
+    width: 24px;
+    height: 24px;
+    left: 93%;
+
+    & .franchise__icon {
+      width: 13px;
+      height: 13px;
+    }
+  }
+}
+
+.franchise__title {
+  display: block;
+  font-family: 'Seymour One';
+  font-weight: 400;
+  font-size: 16px;
+  line-height: calc(24 / 16);
+  text-align: center;
+  color: getColor('theme-black');
+  letter-spacing: 0;
+  width: 221px;
+  height: 72px;
+  margin: 0 auto 20px;
+
+  @include tablet {
+    font-size: 20px;
+    line-height: calc(28 / 20);
+    width: 322px;
+    height: 84px;
+    margin: 0 auto 24px;
+  }
+}
+
+.franchise__text {
+  font-size: 14px;
+  line-height: calc(20 / 14);
+  letter-spacing: 0;
+  color: getColor('theme-black');
+  margin-bottom: 24px;
+  width: 222px;
+  max-height: 240px;
+
+  @include tablet {
+    font-size: 16px;
+    line-height: calc(22 / 16);
+    width: 448px;
+    height: 132px;
+    margin-bottom: 32px;
+  }
+}
+
+.form__wrap {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 24px;
+
+  @include tablet {
+    align-items: center;
+    margin-bottom: 32px;
+  }
+}
+
+.form__input {
+  padding: 8px 12px 7px 12px;
+  outline: none;
+  margin-bottom: 12px;
+  background: rgba(248, 210, 78, 0.1);
+  border: 1px solid #84a178;
+  border-radius: 10px;
+
+  transition: border-color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  width: 224px;
+  height: 32px;
+
+  &:focus {
+    border-color: getColor('orange-main');
+  }
+
+  @include tablet {
+    width: 368px;
+    height: 42px;
+    padding-top: 11px;
+    padding-bottom: 11px;
+    padding-left: 16px;
+  }
+
+  &::placeholder {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: calc(17 / 14);
+    color: getColor('theme-black');
+
+    @include tablet {
+      font-size: 14px;
+      line-height: calc(20 / 14);
+    }
+  }
+}
+
+.form__textarea {
+  resize: none;
+  background: rgba(248, 210, 78, 0.1);
+  border: 1px solid #84a178;
+  border-radius: 10px;
+  padding: 12px 16px;
+  padding: 8px 48px 39px 12px;
+  width: 224px;
+  height: 64px;
+  outline: none;
+  transition: border-color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+  @include tablet {
+    width: 368px;
+    height: 84px;
+    padding-top: 11px;
+    padding-bottom: 53px;
+    padding-left: 16px;
+  }
+
+  &::placeholder {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: calc(17 / 14);
```

Comment:
line-height: calc(17 / 12);

Reply:
&::placeholder - delete
add <label for="name" class="form__label">Name</label> by video Repeta